client/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import AppBar from "@material-ui/core/AppBar";
+import PermIdentityIcon from "@material-ui/icons/PermIdentityOutlined";
+import Toolbar from "@material-ui/core/Toolbar";
+import "./styles.css";
+
+const LoginLink = () => {
+  return (
+    <a id="nav-login" href="#login">
+      Login
+    </a>
+  );
+};
+
+const LogoutLink = () => {
+  return (
+    <a id="nav-logout" href="#logout">
+      Logout
+    </a>
+  );
+};
+
+const ProfileLink = () => {
+  const [name, setName] = useState("Person");
+  return (
+    <div id="nav-profile">
+      <PermIdentityIcon id="nav-profile-icon" />
+      <a id="nav-profile-link" href="#profile">
+        Hi {name}
+      </a>
+    </div>
+  );
+};
+
+const Navbar = () => {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  let rightNav;
```

Comment:
rather than `rightNav`, perhaps we can use a more explicit name like `loginOptions`